bpaotu/bpaotu/views.py

```diff
@@ -593,3 +613,115 @@ def dev_only_ckan_check_permissions(request):
     response = '||'.join([digest, data])
 
     return HttpResponse(response)
+
+
+# --------------------------------------------------------------------------------
+# Add Images to Map
+# --------------------------------------------------------------------------------
+
+# img file extension corresponding to pillow processing format and HttpResponse format
+
+IMG_EXTENSION_TABLE = {
+    'jpg': ['JPEG', 'image/jpeg']
+}
+
+
+LOOKUP_TABLE_KEY = 'lkup_tbl_key'
+
+
+def _get_cached_item(key):
+    try:
+        cache = caches['image_results']
+
+        return cache.get(key)
+    except Exception as e:
+        logger.error(str(e))
+
```

Comment:
don't wrap the base `Exception` and discard, if we are hitting specific exceptions we should deal with those

Reply:
Fixed.